playwright/components/pill/locators.ts

```diff
@@ -0,0 +1,3 @@
+// component preview locators
+export const PILL_PREVIEW = '[data-component="pill"]';
```

Comment:
**suggestion**: you can just use the `pillPreview()` locator I added to the general `index.ts` file instead of creating an index/locator for `Pill` here